site/en/docs/privacy-sandbox/ip-protection/index.md

```diff
@@ -0,0 +1,55 @@
+{% endAside %}
+
+## Implementation status
+
+This document outlines a new proposal to prevent covert tracking: IP Protection.
+
+*  The [IP Protection proposal](https://github.com/spanicker/ip-blindness) has entered [public discussion](https://github.com/spanicker/ip-blindness/issues).
+*  This proposal has not been implemented in any browser.
+*  [The Privacy Sandbox timeline](https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline) provides implementation timings for other Privacy Sandbox proposals. IP Protection is still in its early phase and will have firmer timelines available on this website when available.
+
+## Why do we need this proposal?
+
+IP addresses were created to provide unique identifiers for a client so that traffic can be routed over the internet. IP addresses can be stable over periods of time, which can lead to user identification across first-parties.
+
+IP Protection is a proposal to avoid sharing a user's real IP address with third parties. This proposal suggests using a privacy proxy (previously called a Near-Path NAT) for a connection.  An IP address is an effective cross-site identifier as it is unique, relatively stable, cheap to collect, and IP address collection cannot be detected by the browser. Therefore limiting access to IP addresses is important to prevent methods of cross-site tracking beyond third-party cookies.
+
+## How will IP Protection work?
+
+IP Protection proposes to anonymize the IP address for third parties identified as potentially as using IP addresses for web-wide cross-site tracking.
+ 
+IP Protection proposes a 2-hop [privacy proxy that anonymizes qualifying traffic](https://github.com/spanicker/ip-blindness#privacy-proxy):
+
+* To stop a destination origin from seeing the client’s original IP address.
+* To make sure that proxy and network intermediaries are not privy to the contents of traffic between the client and the destination origin.
+
+### GeoIP
+
+IP-based geolocation is used by a number of services within proxied third-party traffic to obey local laws and regulations. They let services provide content that is relevant to users, such as content localization (for example, to set language), local cache assignment, and geo targeting for ads. To support these needs, the privacy proxy assigns IP addresses that represent the user’s coarse location, including country.
+
+## When will IP Protection be available?
+
+IP protection will not launch as a default setting for Chrome users before 2024. Testing and launch timelines will be informed partly from ecosystem input. The timelines are independent of other anti-covert tracking efforts like phasing out third-party cookies and reducing the information shared by default in the user agent.
```

Comment:
'IP protection will not be available by default for Chrome Stable users before 2024.'